docs/guides/rule_based_profilers/how-to-create-a-new-expectation-suite-using-rule-based-profilers.md

```diff
@@ -0,0 +1,114 @@
+---
+title: How to create a new Expectation Suite using Rule Based Profilers
+---
+import Prerequisites from '../connecting_to_your_data/components/prerequisites.jsx';
+
+<Prerequisites>
+
+- Have configured a Datasource to connect to your data
+- Have a basic understanding of [Metrics in Great Expectations](https://docs.greatexpectations.io/en/latest/reference/core_concepts/metrics.html)
+- Have a basic understanding of [Expectation Configurations in Great Expectations](https://docs.greatexpectations.io/en/latest/reference/core_concepts/expectations/expectations.html#expectation-concepts-domain-and-success-keys)
+
+</Prerequisites>
+
+
+### Profilers
+
+A Profiler uses an Execution Engine to build a new Expectation Suite. It can use zero, one, or more Batches of data to decide which Expectations to include in the new Suite. It can use other information, like a schema or an idea. A profiler may be used to create basic high-level expectations based on a schema even without data, to create specific Expectations based on team conventions or statistical properties in a dataset, or even to generate Expectation Suites specifically designed to be rendered by a Descriptive Renderer for data discovery.
+
+### Rule-Based Profilers
```

Comment:
Should we make any reference to `UserConfigurableProfiler` in this doc? The plan is to ultimately deprecate that object in favor of this YAML config style of rule based profiling right? Should we inform users of the alternative or is that out-of-scope for this doc?

Reply:
This is a great question. My inclination is no, because this is a doc specifically about using Rule-based Profilers, not Profiling generally. I am going to include a link to our general Profiling page though (which itself has a reference to UserConfigurableProfilers.

Reply:
I agree with that reasoning! This page should be about rule based profiling and nothing else. We've got plenty of other resources to cover core GE concepts and profiling as an overall feature.